### ISSUE TITLE
Cordova build error:  strings in switch are not supported in -source 1.6

### ISSUE THREAD

**Issue by nagygergo** [hub9co/cordova-plugin-sntp#1]
Error message:

```
/platforms/android/src/co/hub9/cordova/CDVPluginSntp.java:20: error: strings in switch are not supported in -source 1.6
        switch (action) {
               ^
  (use -source 7 or higher to enable strings in switch)
```

Seems to be some kind of compiler compatibility issue. Using oracle java8 so that shouldn't be the problem. 

An easy fix is to just replace the switch with if-else statements, but that doesn't seem to be a good solution just a hack to get it working.

**Comment by romulof** [comment]
Strings are supported in switch statements since JDK 7

https://docs.oracle.com/javase/8/docs/technotes/guides/language/strings-switch.html

**Comment by nagygergo** [comment]
Yeah, still I looked into some other plugins: https://github.com/sarriaroman/FabricPlugin/blob/master/src/android/FabricPlugin.java

**Comment by nagygergo** [comment]
Yes that's what my problem was at first, but probably cordova-android builds it with other an other version of java. Also for solutions I looked around to check how other cordova plugins solve this problem. The plugins I looked at were doing the method selection this way.

Official apache plugin with the most offered methods for the js interface.
https://github.com/apache/cordova-plugin-globalization/blob/master/src/android/Globalization.java

Some other plugins that do it this way:
https://github.com/sarriaroman/FabricPlugin/blob/master/src/android/FabricPlugin.java

https://github.com/fechanique/cordova-plugin-fcm/blob/master/src/android/FCMPlugin.java

Maybe refactoring back to if-else statements should be considered.
